src/identify.ts

```diff
@@ -3,6 +3,8 @@ export enum IdentifyOperation {
   SET_ONCE = '$setOnce',
   ADD = '$add',
   APPEND = '$append',
+  PREINSERT = '$preInsert',
```

Comment:
Minor, should we name like `PRE_INSERT` as when we use the name in camelcase, we name it `preInsert` above.